athena-teradata/src/main/java/com/amazonaws/athena/connectors/teradata/TeradataMetadataHandler.java

```diff
@@ -349,7 +346,9 @@ private Schema getSchema(Connection jdbcConnection, TableName tableName, Schema
                     String columnName = resultSet.getString("COLUMN_NAME");
 
                     LOGGER.info("Column Name: " + columnName);
-                    LOGGER.info("Column Type: " + columnType.getTypeID());
+                    if (columnType != null) {
+                        LOGGER.info("Column Type: " + columnType.getTypeID());
+                    }
```

Comment:
I think we should log the case where the `columnType` is null, like:

```
else
{
    Logger.warn("Column Type is null for Column Name" + columnName);
}

Reply:
Hi Henry, added the statement.